library/ssl_tls13_generic.c

```diff
@@ -346,7 +346,7 @@ int mbedtls_ssl_write_signature_algorithms_ext( mbedtls_ssl_context *ssl,
     /*
      * Determine length of the signature scheme list
      */
-    for ( md = ssl->conf->sig_hashes; *md != SIGNATURE_NONE; md++ )
```

Comment:
We should rename `md` - it wrongly suggests that (what used to be true to TLS 1.2 hashes) that we're iterating over hash identifiers here.

Reply:
Done